Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;uap10.0;native;net461;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netstandard1.4;uap10.0;native;net461;netcoreapp3.0</TargetFrameworks>
```

Comment:
3.0 is not supported anymore. We should target at least 3.1.

Reply:
@azchohfi Is there any technical reason we need to be targeting 3.1? I don't want to force developers to upgrade their apps just to use notifications, I want as little friction as possible for them. Is there a downside of us keeping the version at 3.0?

I think that'd just add a lot of confusion, especially since if you try to install on a 3.0 app, I assume it'd install the netstandard version which doesn't have the desktop helpers, and you'd be insanely confused why the desktop helpers aren't showing up and why notifications aren't working...

Reply:
3.0 is not supported anymore (EOL). No one should use it, and since dotnet will not support us, we would be in a delicate position to support this on 3.0. Tooling to build this on Azure Devops gets tricky, as we have to install an old SDK.
https://dotnet.microsoft.com/platform/support/policy/dotnet-core
https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3382
https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3383